	Building ICU on the FreeBSD kernel arch included in Debian.
	Reported by Hilmar Preusse <hille42@web.de>.

diff -ur icu-49.rc.orig/source/tools/toolutil/udbgutil.cpp icu-49.rc/source/tools/toolutil/udbgutil.cpp
--- icu-49.rc.orig/source/tools/toolutil/udbgutil.cpp	2012-03-02 23:12:06.000000000 +0100
+++ icu-49.rc/source/tools/toolutil/udbgutil.cpp	2012-03-12 14:27:29.000000000 +0100
@@ -341,7 +341,7 @@
 #if U_PLATFORM_HAS_WIN32_API
     return "Windows";
 #elif U_PLATFORM == U_PF_UNKNOWN
-    return "unknown"
+    return "unknown";
 #else
     return "Other (POSIX-like)";
 #endif
